### PR TITLE
Update 00-release-notes.md

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR updates the release notes with new information from the Edge team.

## Review Changes

💻 [Preview URL](https://deploy-preview-1311--docs-spectrocloud.netlify.app/release-notes)

### CODE REVIEW

File: content/docs/00-release-notes.md
Comment (ritawatson) [inline content/docs/00-release-notes.md:100]:
```suggestion
- Prior Edge Installer versions are incompatible with Palette 3.4 and newer versions due to product enhancements and security fixes. New Edge clusters deployed with an earlier Edge Installer version will not operate in Palette. Active Edge clusters in Palette will continue to work as expected. Use the latest version of the [Edge Installer](/spectro-downloads#edgeinstallerimage) when creating Edge artifacts and deploying new Edge clusters.
```

**Review by ritawatson — APPROVED** [review]
One fix, otherwise LGTM.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1311#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1311#pullrequestreview-1440259705" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
